scripts/updatebom.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+if ! [[ "$0" =~ "scripts/updatebom.sh" ]]; then
+	echo "must be run from repository root"
+	exit 255
+fi
+
+echo "installing 'bill-of-materials.json'"
+go get -v -u github.com/coreos/license-bill-of-materials
+
+# TODO: remove 5 lines from "project": "github.com/coreos/etcd/cmd/etcd"
+license-bill-of-materials \
```

Comment:
Doesn't seem to work with symlinks of symlinks?

```bash
rm -rf ./gopath
mkdir -p ./gopath
ln -s ./cmd/vendor ./gopath/src

GOPATH=`pwd`/gopath license-bill-of-materials \
    --override-file ./bill-of-materials.override.json \
    github.com/coreos/etcd github.com/coreos/etcd/etcdctl
```

Will try other commands as well